node/resolvers/Queries/Users.ts

```diff
@@ -816,6 +816,44 @@ export const getOrganizationsByEmail = async (
   }
 }
 
+export const getOrganizationsByEmailPaginated = async (
```

Comment:
For me, it doesn't make sense to implement the query here, it should be implemented on b2b-organization-graphql.

Reply:
@Rudge the intention of having the query here is to maintain the consistency with the not paginated query that is implemented in the storefront-permissions app as well.